src/smc-webapp/r_misc/close-x.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import { Icon } from "./icon";
+
+const closex_style: React.CSSProperties = {
```

Comment:
Oh, so *this* (use `React.CSSProperties`) is how to declare objects and using them for css.  I've been confused about this.    I should have asked you in chat.